docs/docs/Workspace/workspace-playground.md

```diff
@@ -22,19 +22,16 @@ Notice how the **Playground's** window arrangement changes depending on what com
 :::
 
 
-You can also open a flow's **Playground** without entering its workspace. From **My Collections** or **Langflow Store**, click the **Playground** in one of the projects card.
-
-
 ## Memory Management {#821a5ee2a8a44d35b49943cba630511c}
 
 
 ---
 
-Click the Memories tab to open a table of previous interactions for a specific flow.
+Click on any chat session menu on the right, then click on the message logs option to open a table of previous interactions for a specific flow.
```

Comment:
To view messages by session ID, from the **Playground**, click the **Options** menu of any session, and then select **Message Logs**.